aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapterQueryTest.java

```diff
@@ -64,21 +68,39 @@ public static void enableStrictMode() {
     }
 
     /**
-     * Remove any old database files, and the re-provision a new storage adapter,
+     * Remove any old database files, and then re-provision a new storage adapter,
      * that is able to store the Comment-Blog family of models.
      */
     @Before
     public void setup() {
         TestStorageAdapter.cleanup();
-        this.adapter = TestStorageAdapter.create(AmplifyModelProvider.getInstance());
+        this.blogAdapter = TestStorageAdapter.create(AmplifyModelProvider.getInstance());
+    }
+
+    /**
+     * Remove any old database files, and then re-provision a new storage adapter,
+     * that is able to store the Phone Call family of models.
+     */
+    public void setupForCallModel() {
```

Comment:
Is `blogAdapter` and `callAdapter` being used exclusively in each tests? Or in other words, are they used in same test method?
If not, we can avoid having these two instances of `SynchronousStorageAdapter` and multiple renamings. Cheapest way to do it is to call `tearDown` as first call in `setupForCallModel`
Then initialize `adapter` with the `phonecall.AmplifyModelProvider`.
With that, `teardownForCallModel` can also be removed as it will be same as `tearDown`